lib/vector-buffers/src/variants/disk_v2/writer.rs

```diff
@@ -66,8 +66,6 @@ where
     /// record being big enough that it would exceed the max data file size.
     ///
     /// The record that was given to write is returned.
-    ///
-    /// See lemma 5.
```

Comment:
This is an old comment from when I actually listed out lemmas about the disk buffer design.